InscryptionAPI/Triggers/Interfaces.cs

```diff
@@ -1,240 +1,271 @@
-using System.Collections;
 using DiskCardGame;
+using InscryptionAPI.Helpers;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Text;
 
-namespace InscryptionAPI.Triggers;
-
-/// <summary>
-/// Run when the card this trigger is attached to is added to the hand
-/// </summary>
-public interface IAddedToHand
+namespace InscryptionAPI.Triggers
```

Comment:
It looks like this file was also copied from a local branch

Reply:
is there an issue with this other than me being lazy?